### PR TITLE
Create Patterns doc “empty-states”

### PR DESCRIPTION
**Pull request by dannoye**
Automatically generated by Netlify CMS

### CODE REVIEW

File: src/_areas/design-system/patterns/empty-states.md [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+
+**Text-heavy**
+
+![Empty state large space text](/images/emptystate_largespace_text.svg)
+
+#### Tables
+
+Empty states help guide a user when interacting with an empty table and help them accomplish their next course of action. A title and body text is required within an empty state in a Table.
+
+For detailed documentation on Cards, [view the design specs tables](https://booster.zywave.dev/design-system/components/tables/?tab=usage "https\://booster.zywave.dev/design-system/components/tables/?tab=usage")
```
Comment (p-ob) [inline src/_areas/design-system/patterns/empty-states.md:65]:
```suggestion
For detailed documentation on Tables, [view the Table design specs](https://booster.zywave.dev/design-system/components/tables/?tab=usage "https\://booster.zywave.dev/design-system/components/tables/?tab=usage")
```

File: src/_areas/design-system/patterns/empty-states.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+
+They are most commonly seen the first time a user interacts with a product or page but can also be encountered when data has been deleted or is unavailable.
+
+- - -
+
+## Anatomy
+
+![Empty states anatomy](/images/emptystate_anatomy.svg)
+
+1. **Illustration (optional):** The illustration is used to give the user more insight into what will be available on the page when data has been added or is available. For detailed documentation on our illustrations, view [Booster Illustrations](https://booster.zywave.dev/design-system/visuals/illustrations/ "https\://booster.zywave.dev/design-system/visuals/illustrations/").
```
Comment (juliexiong) [inline src/_areas/design-system/patterns/empty-states.md:22]:
```suggestion
1. **Illustration (optional):** The illustration is used to give the user more insight into what will be available on the page when data has been added or is available. For detailed documentation on our illustrations, view [Booster Illustrations](/design-system/visuals/illustrations/ "/design-system/visuals/illustrations/").
```
Let's use relative URLs in our docs so if we moved to a different domain, we don't have a bunch of broken links.

File: src/_areas/design-system/patterns/empty-states.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+
+- - -
+
+## Layout & structure
+
+The layout and structure of any empty state is determined by application and component context.
+
+#### Cards
+
+Cards are a very common component for holding empty states. Use a hero or spot illustration based on your use case. For detailed documentation on Illustrations, [view the Illustration design specs.](https://booster.zywave.dev/design-system/visuals/illustrations/ "https\://booster.zywave.dev/design-system/visuals/illustrations/")
```
Comment (juliexiong) [inline src/_areas/design-system/patterns/empty-states.md:37]:
```suggestion
Cards are a very common component for holding empty states. Use a hero or spot illustration based on your use case. For detailed documentation on Illustrations, [view the Illustration design specs.](design-system/visuals/illustrations/ "/design-system/visuals/illustrations/")
```

Same thing here.

File: src/_areas/design-system/patterns/empty-states.md [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+
+For detailed documentation on Cards, [view the Card design specs.](https://booster.zywave.dev/design-system/components/cards/ "https\://booster.zywave.dev/design-system/components/cards/")
```
Comment (juliexiong) [inline src/_areas/design-system/patterns/empty-states.md:39]:
```suggestion
For detailed documentation on Cards, [view the Card design specs.](/design-system/components/cards/ "/design-system/components/cards/")
```

And here.

File: src/_areas/design-system/patterns/empty-states.md [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+<tbody>
+
+<tr>
+<td>First use</td>
+<td>Occurs in a new product when the user has not started a workflow. For example:<br>
+
+<li>when starting a new project</li>
+
+<li>user's first time entering a product</li>
+
```
Comment (juliexiong) [inline src/_areas/design-system/patterns/empty-states.md:89]:
```suggestion
<ul>
<li>when starting a new project</li>
<li>user's first time entering a product</li>
</ul>
```
All `<li>`s should be wrapped in `<ul>` for unordered list or `<ol>` for ordered list.

File: src/_areas/design-system/patterns/empty-states.md [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+</td>
+<td>Explain to the user where and how to get started within the new product</td>
+</tr>
+<tr>
+<td>No results data</td>
+<td>Occurs when the user has successfuly completed an action or query but no results or data is returned: For example:<br>
```
Comment (juliexiong) [inline src/_areas/design-system/patterns/empty-states.md:95]:
```suggestion
<td>Occurs when the user has successfuly completed an action or query but no results or data is returned. For example:<br>
```

**Pull request merged**